features/registry/registry.feature

```diff
@@ -106,6 +106,91 @@ Feature: Testing registry
+      | resource: clusterroles                     |
+      | name: registry-registry-role               |
+      | resource: clusterrolebindings              |
+      | name: registry                             |       
+      | resource: serviceaccounts                  |
+      | name: image-registry-certificates          |
+      | resource: configmaps                       |
+      | name: image-registry-private-configuration |
+      | resource: secrets                          | 
+      | name: image-registry                       |
+      | resource: services                         |
+      | name: node-ca                              |
+      | resource: daemonsets                       |
+      | name: image-registry                       |
+      | resource: deployments                      |
+    When I run the :get admin command with:
+      | resource      | clusterroles    |
+      | resource_name | system:registry |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | clusterrolebindings    |
+      | resource_name | registry-registry-role |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | serviceaccounts          |
+      | resource_name | registry                 |
+      | namespace     | openshift-image-registry |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | configmaps                  |
+      | resource_name | image-registry-certificates |
+      | namespace     | openshift-image-registry    |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | secrets                              |
+      | resource_name | image-registry-private-configuration |
+      | namespace     | openshift-image-registry             |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | ds                        |
+      | resource_name | node-ca                   |
+      | namespace     | openshift-image-registry  |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | service                   |
+      | resource_name | image-registry            |
+      | namespace     | openshift-image-registry  |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | service                   |
+      | resource_name | image-registry            |
+      | namespace     | openshift-image-registry  |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | deployments    |
+      | resource_name | image-registry |
+    Then the step should succeed
+    
+  # @author wewang@redhat.com
+  # @case_id OCP-23063
+  @admin
+  Scenario: Check the related log from must-gather tool
+    When I run the :delete admin command with:
```

Comment:
@wewang58 should this be destructive?  Also should a restore step be needed?

Reply:
@pruan-rht just tried, when delete, then co image-registry has no affect, pods are running. operator status is still available, maybe no need restore.

Reply:
@wewang58 I don't know either...is there someone you can check to make sure?

Reply:
@wzheng1 when I do the step: http://pastebin.test.redhat.com/848439, no ds rollout seems, pods are not restart, do you think if need restore step, thanks

Reply:
It seems no any restart, but I suggest to add destructive tag to avoid any operation working on image registry operator in the meantime.

Reply:
@wzheng1 Yes, already added destructive tag, thanks